app/models/invoice.rb

```diff
@@ -39,7 +39,7 @@ class InvalidInvoiceStateException < RuntimeError; end
   has_many :payment_transactions, -> { oldest_first }, dependent: :nullify, inverse_of: :invoice
   has_many :payment_intents, dependent: :destroy, inverse_of: :invoice
 
-  has_attached_file :pdf, url: ':url_root/:class/:id/:attachment/:style/:basename.:extension'
+  has_attached_file :pdf, url: ':url_root/:class/:id/:attachment/:style/:basename.:extension', adapter_options: { hash_digest: Digest::SHA256 }
```

Comment:
For this we need to rehash all attachments which might be an issue for updating existing deployments. See [paperclip docs](https://www.rubydoc.info/gems/paperclip/6.1.0#checksum-fingerprint).

Reply:
Yep, same as above:

```ruby
CLASS=Invoice ATTACHMENT=pdf rake paperclip:refresh:fingerprints
```

Reply:
We are not fingerprinting.